packages/server-wallet/src/wallet/store.ts

```diff
@@ -697,6 +699,13 @@ async function createChannel(
   fundingLedgerChannelId: Bytes32 | undefined,
   txOrKnex: TransactionOrKnex
 ): Promise<Channel> {
+  if (fundingLedgerChannelId) {
+    const ledger = await Channel.forId(fundingLedgerChannelId, txOrKnex);
+    if (!ledger) throw new StoreError(StoreError.reasons.invalidFundingLedgerChannelId);
+    if (ledger.supported && ledger.supported.isFinal)
```

Comment:
It is also possible for the ledger channel to be _closing_, but `ledger.supported.isFinal` to be false.